lib/podlet.js

```diff
@@ -416,6 +425,23 @@ const PodiumPodlet = class PodiumPodlet {
         });
         return result.length === 0 ? '' : result[0];
     }
+
+    [_currentScope](incoming) {
+        const fallback = this.fallback({ prefix: true });
+        const content = this.content({ prefix: true });
```

Comment:
The `prefix` might throw us off here. I don't think there is many which is using `prefix` (because its not intuitive) so I am OK merging this due to the fact that we will remove `prefix` in a future version of Podium.